plugins/modules/aci_context_policy.py

```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_context_policy
+short_description: Manage Route Context Policy (rtcrtl:CtxP)
+description:
+- Manage Route Context Policies for the Route Control Profiles on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+  route_control_profile:
+    description:
+    - Name of an existing route control profile.
+    type: str
+    aliases: [ rtctrl_profile_name ]
+  context_policy:
+    description:
+    - Name of the context profile being created.
+    type: str
+    aliases: [ name, context_name ]
+  action:
+    description:
+    - The action required when the condition is met.
+    type: str
+    choices: [ deny, permit ]
+  action_rule:
+    description:
+    - Name of the action rule profile to be associated with this route context policy.
+    - Set the rules for a Route Map.
+    - See module M(cisco.aci.aci_tenant_action_rule_profile).
```

Comment:
Do we want to do this also for other inputs for which we have a module like tenant, l3out, route_control_profile?

Reply:
No, we should add it to the notes section just like we usually do for other modules.